File: Domain/Portion.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class Portion : BaseEntity<Guid>
    {
        public string Size { get; set; }
        public int Price { get; set; }
        public Dish Dish { get; set; }
    }
}

File: client-app/src/app/models/portion.ts
import { Dish } from "./dish";

export interface Portion {
    id?: string | undefined;
    size: string;
    price: number;
    dish?: Dish;
}

export class Portion implements Portion {
    size: string = '';
    price: number = 0;

    constructor(size: string, price: number) {
        this.size = size;
        this.price = price;
    }
}